<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectComputer</class>
 <widget class="QDialog" name="SelectComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>433</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ВЫБОР КОМПЛЕКТУЮЩИХ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_ram">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>RAM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_gpu">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>GPU</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_cpu">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>CPU</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_storage">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Storage</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_blockPower">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>BP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="comboBox_ram">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_gpu"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_cpu"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_storage"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_blockPower"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit_ram"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_gpu"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_cpu"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_storage"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_blockPower"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="button_selectComputer">
        <property name="text">
         <string>Подобрать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_result">
          <property name="text">
           <string>РЕЗУЛЬТАТ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_computers"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
